### ISSUE TITLE
Support buttons / sliders you can press during the simulation

### ISSUE THREAD

**Issue by EwoutH** [Corvince/mesa-interactive#7]
It would be awesome to have buttons and sliders that you can modify during the simulation. Like triggering some event or policy, or modifying some value to see how the dynamic or balance in the system changes.

In NetLogo:
![Screenshot_291](https://github.com/Corvince/mesa-interactive/assets/15776622/a68de673-4b2b-4d27-bcaa-723504696102)

**Comment by Corvince** [comment]
Yes that would be great. Do you suggest adding them as part of the definition or interactively?

**Comment by EwoutH** [comment]
I think both if fine, I'm slightly leaning towards definition. That way everything is still (defined by) plain Python code.
